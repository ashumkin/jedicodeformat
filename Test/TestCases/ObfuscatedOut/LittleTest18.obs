unit littletest18;interface uses forms;type tfooform=class(TForm)public class function
foo:tfooform;end;implementation type tmonkey=real;class function tfooform.foo:tfooform
;begin result:=nil;end;end.