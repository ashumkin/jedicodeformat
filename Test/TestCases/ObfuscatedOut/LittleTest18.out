unit littletest18;

interface

uses Forms;

type
  tfooform = class(TForm)
  public
    class function foo: tfooform;
  end;

implementation

type
  tmonkey = real;

class function tfooform.foo: tfooform;
begin
  Result := nil;
end;

end.
